packages/terra-doc-template/tests/wdio/doc-template-spec.js

```diff
@@ -1,15 +1,15 @@
 Terra.describeViewports('DocTemplate', ['large'], () => {
   describe('Fully filled out doc', () => {
     before(() => browser.url('/#/raw/tests/terra-doc-template/doc-template/default-doc-template'));
-
+    browser.pause(50);
```

Comment:
these should be in `it` blocks to be executed by the mocha framework

Reply:
![](https://media.giphy.com/media/l2Je7GLhkevHc5I9G/giphy.gif)